source/dem/dem.cc

```diff
@@ -449,6 +451,11 @@ DEMSolver<dim>::particle_wall_contact_force()
   pw_contact_force_object->calculate_pw_contact_force(
     pw_pairs_in_contact, simulation_control->get_time_step(), momentum, force);
 
+  forces_boundary_information[simulation_control->get_step_number()] =
```

Comment:
Don't we need to call these functions only if the calculate_force_torque parameter is true in the parameter handler file?

Reply:
Is that necessary, like the member class would stay at zero tensor, yeah a big map is gonna be created of zero tensor. Would it be  significant?